### ISSUE TITLE
Implement average precision metric to Tensorflow as Callback

### ISSUE THREAD

**Issue by gabrieldernbach** [gabrieldernbach/acoustic_train_classification#22]
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html#sklearn.metrics.average_precision_score

http://www.chioka.in/differences-between-roc-auc-and-pr-auc/

**Comment by gabrieldernbach** [comment]
implemented as of commit 019dba15d418b087619fb57813547fef56f76895
